### ISSUE TITLE
Can not reproduced the results in the paper.

### ISSUE THREAD

**Issue by leoozy** [princeton-nlp/SimCSE#23]
Hello, I tired the run_unsup_example.sh files and can not reproduced the results in your paper.
My environment is : Pytorch 1.8.0 + CUDA 11.1 + NVIDIA V100
the hyperparameters are:

{
  "best_metric": 0.7652876377731959,
  "best_model_checkpoint": "result/my-unsup-simcse-bert-base-uncased",
  "epoch": 0.12794268167860798,
  "global_step": 250,
  "is_hyper_param_search": false,
  "is_local_process_zero": true,
  "is_world_process_zero": true,
  "log_history": [
    {
      "epoch": 0.06,
      "eval_avg_sts": 0.7227670477928896,
      "eval_sickr_spearman": 0.7040163496401507,
      "eval_stsb_spearman": 0.7415177459456286,
      "step": 125
    },
    {
      "epoch": 0.13,
      "eval_avg_sts": 0.7386178016746645,
      "eval_sickr_spearman": 0.7119479655761333,
      "eval_stsb_spearman": 0.7652876377731959,
      "step": 250
    }
  ],
  "max_steps": 1954,
  "num_train_epochs": 1,
  "total_flos": 0,
  "trial_name": null,
  "trial_params": null
}

The eval resutls are:

epoch = 1.0
eval_CR = 86.69
eval_MPQA = 87.75
eval_MR = 81.12
eval_MRPC = 73.53
eval_SST2 = 86.58
eval_SUBJ = 99.6
eval_TREC = 82.67
eval_avg_sts = 0.7371936168707977
eval_avg_transfer = 85.41999999999999
eval_sickr_spearman = 0.7082728178184948
eval_stsb_spearman = 0.7661144159231005
But the results in your paper are:

![image](https://user-images.githubusercontent.com/26025961/117586503-2ff23c80-b14b-11eb-87d2-99a515971369.png)

**Comment by gaotianyu1350** [comment]
Hi,

Thanks for your feedback! The reported numbers in the training process are "development set results", while numbers in the table are the "test set results". To get the test set results, you should follow the instructions in README and use `evaluation.py`.

Besides, as we mentioned in the README, different computing devices may lead to slightly different results (we use Nvidia 3090 GPUs).

**Comment by leoozy** [comment]
I am sorry for my mistakes. I will try it.
